userspace/libsinsp/container_engine/cri.cpp

```diff
@@ -191,9 +198,15 @@ cri::cri(container_cache_interface &cache) : container_engine_base(cache)
 {
 	if (s_cri_unix_socket_paths.empty())
 	{
-		// Default value when empty
+		// containerd as primary default value when empty
 		s_cri_unix_socket_paths.emplace_back("/run/containerd/containerd.sock");
 	}
+	if (s_cri_unix_socket_paths.empty())
```

Comment:
@gnosek almost forgot to ask, right now this is unreachable and I only used it for more convenient dev when commenting the other one out. Why don't we check if the socket even exists here and then add defaults?

Reply:
We used to support just one socket here (it used to be just a  `std::string s_cri_unix_socket_path` IIRC) so praying and setting the default to containerd kind of made sense ;)

But still, we check for socket existence just below, so it may be enough to add both sockets in one `if(empty())` block and let the code below sort it out

Reply:
Done ;)